css/reset.css

```diff
@@ -0,0 +1,74 @@
+* {
+  padding: 0;
+  margin: 0;
+  border: 0;
+}
+*,
+*:before,
+*:after {
+  -moz-box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+}
+:focus,
+:active {
+  outline: none;
+}
+a:focus,
+a:active {
+  outline: none;
+}
+nav,
+footer,
+header,
+aside {
+  display: block;
+}
+html,
+body {
+  height: 100%;
+  width: 100%;
+  font-size: 100%;
+  line-height: 1;
+  font-size: 14px;
+  -ms-text-size-adjust: 100%;
+  -moz-text-size-adjust: 100%;
+  -webkit-text-size-adjust: 100%;
+}
+input,
+button,
+textarea {
+  font-family: inherit;
```

Comment:
if you set font-family for *{}
than you don't need to set it for each element
in this case font-family rule is needed only for elements. that has another font

Reply:
so i delete this and add font-family: inherit to *{}?

Reply:
actually I don't understand why you need this rule with value inherit
some rule adds then, when you have wrong behaviour
do you have wrong behavior for font-family for this items?
input,
button,
textarea